src/DefaultBuilder/src/Configuration.cs

```diff
@@ -19,69 +19,105 @@ namespace Microsoft.AspNetCore.Builder
     public sealed class Configuration : IConfigurationRoot, IConfigurationBuilder, IDisposable
```

Comment:
Side note: just when I thought we no longer owned configuration :)  This seems like a decent candidate for at least inclusion in the extensions configuration packages (if not an easy way to make this the default configuration root), is the idea that we bake this and eventually push this down to extensions.config in a future release?

Reply:
Plan is to do that for this release